app/controllers/activities/ical.js

```
@@ -49,4 +47,23 @@ export default class ActivitiesIcalController extends Controller {
+    const { currentUser } = this.session;
+    const selectedCategories = this.categoriesParams;
+
+    currentUser.set('icalCategories', selectedCategories);
+
+    return currentUser
+      .save()
+      .then(() => {
+        this.flashNotice.sendSuccess('Categorieën succesvol opgeslagen!');
+      })
+      .catch((error) => {
+       this.flashNotice.sendError(
```

ESLint

[error] Insert `·`

(prettier/prettier)